dashboard/db/migrate/20160426210000_add_levels_to_script_level.rb

```diff
@@ -0,0 +1,33 @@
+class AddLevelsToScriptLevel < ActiveRecord::Migration
+  def up
+    create_join_table :levels, :script_levels do |t|
+      t.index :level_id
+      t.index :script_level_id
+    end
+
+    ScriptLevel.class_eval do
+      belongs_to :old_level, class_name: "Level", foreign_key: "level_id"
+    end
+
+    ScriptLevel.all.each do |scriptlevel|
+      unless scriptlevel.old_level.nil?
+        scriptlevel.levels << scriptlevel.old_level
+        scriptlevel.save
```

Comment:
Here and below I think it would be worth using save! to throw an exception and abort the migration if this somehow doesn't work (unlikely, but we would definitely want to know)